packages/paste-website/src/components/Roadmap/Roadmap.tsx

```diff
@@ -47,14 +47,14 @@ const Roadmap: React.FC<RoadmapProps> = ({data}) => {
               <Heading as="h2" variant="heading20">
                 <Box as="span" display="flex" alignItems="center">
                   Release - {release.release}
-                  <Anchor data-cy="anchored-heading-h2" href={`#${releaseSlug}`}>
+                  <StyledAnchorHyperlink data-cy="anchored-heading-h2" href={`#${releaseSlug}`}>
```

Comment:
Using the StyledAnchorHyperlink in Roadmap too for consistency

Reply:
I actually wonder if we can replace this whole heading block with `AnchoredHeading`? It's basically doing the same thing, but with better markup.

You'd have to do a little bit of moving stuff around, and make sure the `releaseSlug` matches what `AnchoredHeading` produces, but I reckon its doable. Then we're really being consistent accross pages

Reply:
o0o0o interesting, lemme try and see!

Reply:
Made this work; roadmap is now using `AnchoredHeading`